src/entities/SecurityToken/SubModule.ts

```diff
@@ -1,11 +1,20 @@
 import { SecurityToken } from './SecurityToken';
 import { Context } from '../../Context';
 
+/**
+ * SubModule implementation used to references ST and Context
```

Comment:
Represents a namespace inside the Security Token that handles a set of related functionality